dev/com.ibm.ws.beanvalidation.v20/src/com/ibm/ws/beanvalidation/v20/Validation20ClassLoader.java

```diff
@@ -0,0 +1,120 @@
+package com.ibm.ws.beanvalidation.v20;
```

Comment:
we do we have 2 copies of `Validation20ClassLoader`?  Can the one over in beanvalidation.core be removed?

Reply:
also missing copyright

Reply:
beanvalidation.core version has been removed, and the copyright has been added.